### PR TITLE
Week 2

### PR DESCRIPTION
**Pull request by jaimehisao**
Finished the V1 of the portfolio. Basic HTML design but planning on making it pretty after basic code is approved.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,15 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts = ["My Google Drive contains over 22TB of files",
+   "I'm currently watching The Office for the third time",
+    "I have a turtle as a pet but I haven't seen it in a week"];
```
Comment (anniefu) [inline portfolio/src/main/webapp/script.js:21]: are you sure you still have a turtle as a pet, then?

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -1,9 +1,52 @@
...
+  text-align:center;
+  display: flex;
+}
+
+.column {
+  text-align:center;
+  flex: 50%;
+  padding: 5px;
+}
+*/
```
Comment (anniefu) [inline portfolio/src/main/webapp/style.css]: [nit] It's good practice to remove unused/uncomment code. Things like that tend to linger for years.

**Review by anniefu — APPROVED** [review]

File: cloudbuild.yaml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+steps:
```
Comment (anniefu) [inline cloudbuild.yaml:1]: I would actually split this out into its own git branch/PR, because it's mostly unrelated to the rest of the changes here.

Comment (ssmall) [reply cloudbuild.yaml:1]: +1 to this suggestion

File: portfolio/cloudbuild.yaml [hunk 4]
```diff
@@ -0,0 +1,4 @@
+steps:
+ - name: maven:3-jdk-8
+   entrypoint: mvn
+   args: ["package" "appengine:deploy"]
```
Comment (ssmall) [inline portfolio/cloudbuild.yaml]: Is this a duplicate of the file included above?

File: portfolio/src/main/webapp/index.html [hunk 5]
```diff
@@ -1,18 +1,88 @@
...
+          </div>
+        </div>
+        <!--
+        <div class="column">
+          <div class="imgcontainer">
+            <a href="https://mx.linkedin.com/in/jaime-hisao-yesaki-750a02155" target="_blank"><img class="imglink"
+                src="/images/lin.png" alt="Logo de LinkedIn" width="60" height="60"></img></a>
+          </div>
+        </div>
+        -->
```
Comment (ssmall) [inline portfolio/src/main/webapp/index.html]: nit: can this be removed?

**Review by ssmall — CHANGES_REQUESTED** [review]
Over all looks great! Just a couple comments/questions :)

File: cloudbuild.yaml (hunk 3 above)
Comment (jaimehisao) [reply cloudbuild.yaml:1]: will do! got exited to quickly implement it and didn't see its in the same PR

File: portfolio/cloudbuild.yaml (hunk 4 above)
Comment (jaimehisao) [reply portfolio/cloudbuild.yaml]: I moved it from the root to the portfolio directory

File: portfolio/src/main/webapp/index.html (hunk 5 above)
Comment (jaimehisao) [reply portfolio/src/main/webapp/index.html]: Was planning on implementing the LinkedIn button, but it messed up my alignment. I was planning on implementing it tomorrow, but submit my progress. This also relates to @anniefu 's comment about on the CSS file

File: portfolio/cloudbuild.yaml (hunk 4 above)
Comment (ssmall) [reply portfolio/cloudbuild.yaml]: It looks like this file and `portfolio/cloudbuild.yaml` are still included in this PR. is that intentional?

**Review by ssmall — COMMENTED** [review]
Looks like both of the `cloudbuild.yaml` files are still included in this PR -- is that intentional?

File: portfolio/cloudbuild.yaml (hunk 4 above)
Comment (jaimehisao) [reply portfolio/cloudbuild.yaml]: No, I've gone ahead and removed it

**Review by ssmall — APPROVED** [review]
```
    __|__ |___| |\
    |o__| |___| | \
    |___| |___| |o \
   _|___| |___| |__o\
  /...\_____|___|____\_/
  \   o * o * * o o  /
~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Ship it!

**Pull request merged**